builds/e2e/longhaul.yaml

```diff
@@ -346,4 +346,69 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.amd64)'
-         
\ No newline at end of file
+
+################################################################################
+  - job: iotuap_amd64_moby
+################################################################################
+    displayName: Windows IoT Core AMD64 Moby
+    condition: eq(variables['run.iotuap.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
+        #- longhaul_iotuap_x64 -equals true
+        - run-long-haul -equals true
+        - runner-os-name -equals WinIoTCore_x64
+    variables:
+      edgelet.artifact.name: 'iotedged-windows'
+    steps:
+      - checkout: none
+      - task: AzureKeyVault@1
+        displayName: 'Access Key Vault Secrets'
+        inputs:
+          azureSubscription: $(azure.subscription)
+          KeyVaultName: $(azure.keyVault)
+          SecretsFilter: 'edgebuilds-azurecr-io-username,edgebuilds-azurecr-io-pwd,IotHubStressConnString,EventHubStressConnStr,StorageAccountMasterKeyStress,SnitchLongHaulAlertUrl'
+      - task: DownloadBuildArtifacts@0
+        displayName: 'Download Edgelet Artifacts'
+        inputs:
+          buildType: specific
+          project: $(vsts.project)
+          pipeline: $(edgelet.package.build)
+          branchName: $(edgelet.package.branchName)
+          buildVersionToDownload: latestFromBranch
+          downloadPath: '$(Build.StagingDirectory)'
+          artifactName: $(edgelet.artifact.name)
+      - task: DownloadBuildArtifacts@0
+        displayName: 'Download Images Artifacts'
+        inputs:
+          buildType: specific
+          project: $(vsts.project)
+          pipeline: $(images.build)
+          branchName: $(images.branchName)
+          buildVersionToDownload: latestFromBranch
+          downloadPath: '$(Build.StagingDirectory)'
+          artifactName: $(images.artifact.name.windows)
+          itemPattern: |
+            $(images.artifact.name.windows)/IotEdgeQuickstart/x64/*
+            $(images.artifact.name.windows)/e2e_deployment_files/long_haul_deployment.template.windows.json
+            $(images.artifact.name.windows)/scripts/windows/setup/IotEdgeSecurityDaemon.ps1
+            $(images.artifact.name.windows)/scripts/windows/test/Run-E2ETest.ps1
+            $(images.artifact.name.windows)/scripts/windows/test/Setup-Env.ps1
```

Comment:
Please update another 2 windows section to download Setup-Env.ps1 file in long haul, stress and e2e tests.

Reply:
Do you want to do this as part of a separate PR for Windows Pro and Server and keep this one just focused on Windows IoT Core?

Reply:
Run-E2ETests.ps1 is shared; it will break e2e tests on other Windows Platform, right?

Reply:
Done, added Setup-Env.ps1 to the list of artifact items to be copied for Windows Pro and Server too.